### ISSUE TITLE
Add `bowtie report` for automatically uploading a report to the UI

### ISSUE THREAD

**Issue by Julian** [bowtie-json-schema/bowtie#696]
We should make:

`bowtie suite -i foo 7 | bowtie report` automatically open a web browser with the report uploaded to it, using our [local report page](https://bowtie.report/#/local-report/).

(n.b. we used to have a CLI command called this, before our React rewrite, which did something slightly similar, but we can do this much better now that we have our fancy UI!)

**Comment by sudo-jarvis** [comment]
Hi @Julian, I would like to work on this issue.

**Comment by Julian** [comment]
That would be great -- do you know a way to make this work? When I took a quick look it seemed like we'd need to do it in a headless browser, but if you can make it work, great.

**Comment by sudo-jarvis** [comment]
Yeah ig we can use some library to automate the uploading process. Then this command would simply spawn the browser and the library would automatically upload the generated report so that the user would be able to directly see the results in his browser

**Comment by Julian** [comment]
As far as I saw, the "this command would simply spawn the browser and the library would automatically upload the generated report" part won't work as for security reasons you can't control (run JS) in browser windows unless the entire Browser window is opened in a way that supports selenium/playright manipulating (meaning to do this we'd have to open a separate Browser instance entirely, and at that point it's not worth doing this over just having someone upload the report themselves) -- but give it a shot, maybe I'm wrong.

**Comment by sudo-jarvis** [comment]
Yeah you are right, it wont be able to use the same normal browser instance that the user uses. Any tool that might be used would create its own browser instance

**Comment by Julian** [comment]
Yeah I think that's less interesting which is why I marked this low-priority, but still possibly slightly useful if you want to try working on it.
